ci/jenkins/pipelines/prs/helpers/pr-manager

```diff
@@ -18,9 +18,10 @@ setup_python_env() {
     source ${VENVDIR}/bin/activate
     # Not calling to `pip3` directly because its shebang can exceed 128 characters
     # and this operation will fail. Check: https://github.com/pypa/pip/issues/1773
-    python -m pip install -r "${SDIR}/pr_manager/requirements.txt"
+    python -m pip -q --disable-pip-version-check install -U -r "${SDIR}/pr_manager/requirements.txt"
```

Comment:
I'm not sure I understand why this change was needed.

Reply:
I guess `-q` and `--disable-pip-version-check` is to avoid getting unnecessary output into the log?

Reply:
right. but that is not what this PR is about nor what the commit message talks about. I like to keep our PR's focused on what they said they changed. it makes going back though history easier.

Reply:
Thats a really good point indeed, unrelated changes in the same commit/PR make baby yoda cry :cry:

Reply:
At least, they should be in separated commits,